### ISSUE TITLE
Image manipulation tools are broken (resize etc) 3.5/3.6a

### ISSUE THREAD

**Issue by twij** [pyrocms/pyrocms#4896]
**Describe the bug**
The Intervention image tools seem to be broken and do strange things. Resize with just a width doesn't keep the aspect ratio so distorts images. Using cropped also seems to return a class rather than the decorated object and is missing some related attributes. Works fine in 3.4.

Swapping the Anomaly\Streams\Platform\Image\Image class for the old one seems to be a drop in replacement and you can see that some of the classes changed in these with diff. I presume this must break something else somewhere though. 

Basically:

'if (array_pop($arguments) == true) {' was changed from 'if (array_pop($arguments) !== false && (is_null($arguments[0]) || is_null($arguments[1]))) {'

This seems to fail as it returns null. I presume the other weird behaviour is related to this, but I didn't dig much further than that.

**To Reproduce**
1. {{ page.image.resize(300) }}
2. {{ dd(page.image.cropped.width) }}

Thank you!

**Comment by RyanThompson** [comment]
Just fixed this: https://github.com/anomalylabs/streams-platform/commit/3dd1098da188d7ad41c3a7147b2140c48db9ca65
